src/app/frontend/chrome/chrome.scss

```diff
@@ -22,6 +22,7 @@
 
 .kd-toolbar {
   box-shadow: $whiteframe-shadow-z1;
+  min-height: 64px;
```

Comment:
This shows my issue.
![media](https://cloud.githubusercontent.com/assets/2285385/11956951/70f00d98-a8be-11e5-8174-b07fe45d1284.gif)